### ISSUE TITLE
Signal 11 Animation-related crash

### ISSUE THREAD

**Issue by GTink911** [godotengine/godot#95723]
### Tested versions

- Reproduced on v4.3.stable.mono.official [77dcf97d8]
- Reproduced on v4.3.stable.official [77dcf97d8]
- Reproduced on a self-built version of the latest source, [1bd740d]
- Reproduced on v4.0.stable.official [92bee43ad]

### System information

Godot v4.3.stable.mono - Windows 10.0.22631 - GLES3 (Compatibility) - NVIDIA GeForce GTX 1650 (NVIDIA; 31.0.15.1700) - AMD Ryzen 5 4600H with Radeon Graphics (12 Threads)

### Issue description

For an animation-related reason I cannot discern, this scene (and some others in the project it originated in, which I believe is tied to the presence of the moving platform, though the crash doesn't occur in the scene file dedicated to it) causes the editor to completely crash. When running from the terminal with --verbose, the error `CrashHandlerException: Program crashed with signal 11` appears. (I couldn't find information about how to setup the engine for debugging, so the backtrace is empty and therefore not included.) However, some amount of the time, instead of a crash the error `ERROR: Condition "!has_animation(p_name)" is true.` occurs.

I've reproduced this error on two separate computers using the MRP below, one of which had never had Godot installed on it before prior to me testing it, so I'm fairly confident it's not isolated to me.

If there's any other information I can provide or steps I can take to help with finding the issue, please let me know :)

### Steps to reproduce

1. Download the MRP below
2. Launch the project from the Project Management screen
3. Typically (>60% of the time) the crash will immediately be triggered. If it doesn't, instead the following error will occur:
```
ERROR: Condition "!has_animation(p_name)" is true.
   at: make_animation_instance (scene/animation/animation_mixer.cpp:1909)
```
4. If this happens, restart the editor until the crash happens. I've never had it take more than 5 restarts, and the vast majority of the time it takes less than 3.

### Minimal reproduction project (MRP)

[bug repro.zip](https://github.com/user-attachments/files/16646905/bug.repro.zip)
(.godot folder is included because I've found it to make reproducing the crash more consistent.)

**Comment by TokageItLab** [comment]
Does this occur in 4.2 or earlier? If it is related to #80715, please check to see if it is fixed by #95605.

**Comment by GTink911** [comment]
Occurs on 4.0. Seems like the same problem as that issue and as far as I can tell it is fixed by that PR, will close as duplicate.
